core/test_network/Cargo.toml

```diff
@@ -10,17 +10,21 @@ edition = "2021"
 query = []
 
 [dependencies]
-iroha_core = { version = "=2.0.0-pre-rc.5", path = "../../core" }
+iroha = { path = "../../cli", features = ["test-network"] }
+iroha_actor = { version = "=2.0.0-pre-rc.5", path = "../../actor" }
 iroha_client = { version = "=2.0.0-pre-rc.5", path = "../../client" }
+iroha_core = { version = "=2.0.0-pre-rc.5", path = "../../core" }
 iroha_data_model = { version = "=2.0.0-pre-rc.5", path = "../../data_model" }
 iroha_logger = { version = "=2.0.0-pre-rc.5", path = "../../logger" }
-iroha_actor = { version = "=2.0.0-pre-rc.5", path = "../../actor" }
-iroha = { path = "../../cli", features = ["test-network"] }
 iroha_permissions_validators = { version = "=2.0.0-pre-rc.5", path = "../../permissions_validators" }
 
 eyre = "0.6.5"
+futures = { version = "0.3.17", default-features = false, features = ["std", "async-await"] }
+once_cell = "1.8.0"
+rand = "0.8"
 tempfile = "3"
-unique_port = "0.1.0"
 tokio = { version = "1.6.0", features = ["rt", "rt-multi-thread", "macros"]}
-rand = "0.8"
-futures = { version = "0.3.17", default-features = false, features = ["std", "async-await"] }
+unique_port = { version = "0.2.1", git = "https://github.com/ales-tsurko/iroha2-unique_port", rev = "09e95bb3608626c995b8e7e51c64ba7486c88948" }
```

Comment:
Shouldn't the `unique_port` upstream be updated and used as a dependency?

Reply:
No. We need to verify that it works first, and updating it quickly (without a PR) is only possible if it uses a personal fork. Before merging this needs to be PR'd into `unique_port` and this can go back to being the actual soramitsu repo.